data/migrations/24.lua

```diff
@@ -1,3 +1,6 @@
 function onUpdateDatabase()
-	return false
+	print("> Updating database to version 25 (change type of `player_storage`.`value` from int to string)")
+
+	db.query("ALTER TABLE `player_storage` CHANGE `value` `value` VARCHAR(255) NOT NULL DEFAULT '0'")
```

Comment:
I don't think storing all values as a string is a good idea. What if a user wants to store "123" as a string? Your code will force that value into an integer after a reload.

Reply:
Yes, it will perform `value = stoi(strValue);` for each storage entry, but it will do it once during user connection. Would a better idea be an additional table dedicated to string storages?